models/brats_mri_segmentation/configs/multi_gpu_train.json

```diff
@@ -24,13 +24,17 @@
     },
     "validate#dataloader#sampler": "@validate#sampler",
     "validate#evaluator#val_handlers": "$None if dist.get_rank() > 0 else @validate#handlers",
-    "training": [
+    "initialize": [
         "$import torch.distributed as dist",
         "$dist.init_process_group(backend='nccl')",
```

Comment:
Please update this referring to the spleen bundle:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/configs/multi_gpu_train.json#L29

Thanks.